setup.py

```diff
@@ -25,7 +25,11 @@
 uwsgi_max_extras = {"uwsgi==2.0.*"}
 uwsgi_min_extras = {"uwsgi==2.0.14"}
 gunicorn_min_extras = {"gunicorn==0.16.1"}
-pyramid_extras = {"pyramid<2", "waitress<2.1"} | trigger_extras
+pyramid_extras = {
+    "pyramid<2; python_version < '3.6'",
+    "pyramid==2.0; python_version >= '3.6'",
```

Comment:
can you make this `pyramid<3`? that's generally been our strategy for framework versioning

Reply:
Yes I'll change that, I made it that way simply because Pyramid 2.0 doesn't support python 2 anymore

Reply:
oh, definitely keep it for python < 3.6, but for the >= 3.6 case you can use pyramid<3

Reply:
Okay thank you